configs/platforms/el-8-aarch64.rb

```diff
@@ -0,0 +1,15 @@
+platform 'el-8-aarch64' do |plat|
+  plat.servicedir '/usr/lib/systemd/system'
+  plat.defaultdir '/etc/sysconfig'
+  plat.servicetype 'systemd'
+
+  packages = %w[
+    autoconf automake createrepo gcc gcc-c++ java-1.8.0-openjdk-devel
+    libsepol libsepol-devel libselinux-devel make pkgconfig cmake readline-devel
+    rsync rpm-build rpm-libs rpm-sign rpmdevtools swig yum-utils zlib-devel
+    systemtap-sdt-devel
```

Comment:
I don't think we need al of this build packages, like `java-1.8.0-openjdk-devel rpm-build rpm-libs rpm-sign rpmdevtools` and might be others too